pipelines/jobs/Semeru_configuration/jdk11u_pipeline_config.groovy

```diff
@@ -222,76 +222,42 @@ class Config11 {
-                        'sanity.perf',
-                        'sanity.jck',
-                        'sanity.system',
-                        'extended.functional',
-                        'extended.openjdk',
-                        'extended.perf',
-                        'extended.jck',
-                        'extended.system',
-                        'special.functional',
-                        'special.jck',
-                        'special.openjdk',
-                        'special.system',
-                        'dev.functional',
-                        'sanity.external',
-                        'dev.external',
-                        'dev.openjdk',
-                        'sanity.jck.fips140_2',
-                        'extended.jck.fips140_2',
-                        'special.jck.fips140_2',
-                        'sanity.openjdk.fips140_2',
-                        'extended.openjdk.fips140_2',
-                        'extended.functional.fips140_3_OpenJCEPlusFIPS',
-                        'sanity.jck.fips140_3_OpenJCEPlusFIPS',
-                        'extended.jck.fips140_3_OpenJCEPlusFIPS',
-                        'special.jck.fips140_3_OpenJCEPlusFIPS',
-                        'sanity.openjdk.fips140_3_OpenJCEPlusFIPS',
-                        'extended.openjdk.fips140_3_OpenJCEPlusFIPS',
-                        'extended.functional.fips140_3_OpenJCEPlusFIPS.FIPS140-3',
-                        'sanity.jck.fips140_3_OpenJCEPlusFIPS.FIPS140-3',
-                        'extended.jck.fips140_3_OpenJCEPlusFIPS.FIPS140-3',
-                        'special.jck.fips140_3_OpenJCEPlusFIPS.FIPS140-3',
-                        'sanity.openjdk.fips140_3_OpenJCEPlusFIPS.FIPS140-3',
-                        'extended.openjdk.fips140_3_OpenJCEPlusFIPS.FIPS140-3'
-                    ],
-                    release : [
-                        'sanity.functional',
-                        'sanity.openjdk',
-                        'sanity.perf',
-                        'sanity.jck',
-                        'sanity.system',
-                        'extended.functional',
-                        'extended.openjdk',
-                        'extended.perf',
-                        'extended.jck',
-                        'extended.system',
-                        'special.functional',
-                        'special.jck',
-                        'special.openjdk',
-                        'special.system',
-                        'dev.external',
-                        'sanity.jck.fips140_2',
-                        'extended.jck.fips140_2',
-                        'special.jck.fips140_2',
-                        'sanity.openjdk.fips140_2',
-                        'extended.openjdk.fips140_2',
-                        'sanity.openjdk.fips140_3_OpenJCEPlusFIPS',
-                        'extended.openjdk.fips140_3_OpenJCEPlusFIPS'
+                        "sanity.external",
```

Comment:
I wonder if we should arrange this list so the extra targets are listed first and then all the default ones after? I also wonder if we should rewrite the code to be "additional test targets" rather than an override list. :thinking: 
If nothing else, I'm listing the diff here to look back on if needed.

From what I see, extra targets:
- sanity.external
- sanity.jck.fips140_2
- sanity.openjdk.fips140_2
- extended.jck.fips140_2
- extended.openjdk.fips140_2
- special.jck.fips140_2
- dev.external